.github/ISSUE_TEMPLATE.md

```diff
@@ -1,80 +1,38 @@
-## 1. General summary of the issue
-
 <!--
 
-Please provide a descriptive title of the issue in the field 'Title' too.
+Please provide a descriptive title in the field 'Title' too.
 
-Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore.
+Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore. Please update Pester to the latest version. See [Installation and update guide](https://pester.dev/docs/introduction/installation).
```

Comment:
I would rephrase this to something like Try updating to the latest version of Pester. Only Pester 4.10.x and 5.x.x are supported, try updating to the latest version to see if that solves your problem.

The current phrasing lists a lot of details, that most people won't care about.